tests/e2e/app-template/config/admin.js

```diff
@@ -14,4 +14,16 @@ module.exports = ({ env }) => ({
     nps: env.bool('FLAG_NPS', true),
     promoteEE: env.bool('FLAG_PROMOTE_EE', true),
   },
+  preview: {
```

Comment:
Is this new? Should we document it somewhere?

Reply:
No it's not documented yet but we're in contact with @pwizla to have that ready for launch. Currently it's also under an undocumented future flag until it's ready